core/src/cache.rs

```diff
@@ -1124,13 +1122,14 @@ impl Cache {
 
     /// Get a reference to the underlying files. Required by
     /// the WASM REPL error reporting code and LSP functions.
-    pub fn files(&self) -> &Files<String> {
+    pub fn files(&self) -> &Files {
         &self.files
     }
 
     /// Get a mutable reference to the underlying files. Required by
     /// [crate::error::IntoDiagnostics::into_diagnostics].
-    pub fn files_mut(&mut self) -> &mut Files<String> {
+    /// TODO: maybe we can delete this now?
```

Comment:
One way to know is to delete it and see if that breaks :stuck_out_tongue: I'm in favor of deleting it if it's not useful anymore. It's a public item and honestly it's a bit ugly.

Reply:
Ah, I was hoping to notice the "TODO" before making this pr :sweat_smile: I guess I'll need to find a more unique string...

And yes, we can delete it and clean up a few things.